include/solvers/navier_stokes_scratch_data.h

```diff
@@ -723,57 +703,30 @@ class NavierStokesScratchData
               undisturbed_flow_force[d]                                   = 0.;
             }
 
-          fluid_velocity_at_particle_location[particle_no] = 0;
-
           // Stock the values of particle velocity in a tensor
-          particle_velocity[particle_no][0] =
+          particle_velocity[i_particle][0] =
             particle_properties[DEM::PropertiesIndex::v_x];
-          particle_velocity[particle_no][1] =
+          particle_velocity[i_particle][1] =
             particle_properties[DEM::PropertiesIndex::v_y];
-          if (dim == 3)
-            particle_velocity[particle_no][2] =
+          if constexpr (dim == 3)
+            particle_velocity[i_particle][2] =
               particle_properties[DEM::PropertiesIndex::v_z];
 
-          // Interpolate velocity and void fraction at particle position
-          // Reference location of the particle
-          auto reference_location = particle.get_reference_location();
+          cell_void_fraction[i_particle] = 0;
+          if (!interpolated_void_fraction)
```

Comment:
Do we still use this? I think we could remove interpolated void fraction = false.

Reply:
I have implemented this for testing purposes, but I don't think it is necessary

Reply:
You or I can make another PR to remove the feature afterwards. I just feel that removing features require PR on its own because of the documentation change and the CHANGELOG. This way it is clearer.